documentation/examples/authentication/system_authentication.ts

```diff
@@ -0,0 +1,11 @@
+import * as coda from '../../../index';
+
+const pack = coda.newPack();
+
+// BEGIN
+
+pack.setSystemAuthentication({
+  // Replace HeaderBearerToken with an authentication type
+  // besides OAuth2, CodaApiHeaderBearerToken, None and Various.
```

Comment:
Nit, would simplify this second line to something like "(Not all authentication types are available for system authentication.)"